### ISSUE TITLE
[FR] Excalidraw link support

### ISSUE THREAD

**Issue by Dennyyusuf** [RyotaUshio/obsidian-pdf-plus#138]
### Describe your request

Hello Developers!

Thanks for your great work for this amazing plugin, your effort and work is immensely valuable for obsidian community.
I have one FR:

I notice that once we copied the highlighted pdf we are not getting the same result with pasting it on Excalidraw as in the markdown file (it s unable to deep linking in excalidraw), esp when both files are open 

<img width="1000" alt="image" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/65077044/a147714a-45f5-4be4-bd86-be881d4b6db0">

when we clicked to pdf files it also unable to pinpoint the excalidraw pasted highlight text and gives below notif
<img width="271" alt="image" src="https://github.com/RyotaUshio/obsidian-pdf-plus/assets/65077044/0041463f-9f1a-4dd8-a752-ad58795f44a4">

It would be great if we can enhance this pdf++ experience by having visual representative of our pdf notes inside excalidraw canvas will full functionality.

Thanks again for this great plugin. 

Cheers,
Denny

**Comment by RyotaUshio** [comment]
> it s unable to deep linking in excalidraw

What exactly do you mean here?

I quickly tested, and if I paste a link to a text selection into an Excalidraw canvas and save the change, the text was successfully highlighted:

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/2cc61c16-864c-49f0-bf51-d33736a2bc86

If I click on the link, I can jump to the exact location of the text selection (it opens the same PDF file in a new tab, which is not great, but I cannot do anything since it's governed by Excalidraw, not PDF++).

Also, if I cmd+hover over the highlight, I can preview the link in the Excalidraw file:

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/8717bd29-770d-413c-ad76-f200b49af7e6

Double-clicking it can take me to the location of the link in the Excalidraw file:

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/66521111-5a0f-4a83-b77c-ce53c22de654

The only thing I found weird is that Excalidraw _sometimes, not always,_ displays the notification "I could not find a matching text element" when I do the same thing with the Excalidraw file already opened:

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/72342591/77c66f70-b0a4-4a45-9503-3c80bd635918

**Comment by Dennyyusuf** [comment]
Yes I believe there is something need to be tweaked at Excalidraw side..
For me when I annotate more than one highlight then when I click at excalidraw it can't pinpoint the location of the highlight inside the pdf

https://github.com/RyotaUshio/obsidian-pdf-plus/assets/65077044/ee29b71d-2227-42cc-9fe2-c53b130e8cea
Thansks for your time!

**Comment by Rex-Diego** [comment]
I find when using Excalidraw in a new window, the backlink can bring you back to PDF++ in another window in a correct way. Another benefit is that both plugins will be faster than they stay in the same window.

**Comment by Dennyyusuf** [comment]
> I find when using Excalidraw in a new window, the backlink can bring you back to PDF++ in another window in a correct way. Another benefit is that both plugins will be faster than they stay in the same window.

Hmm weird, I am still unable to get this deep linking even when I followed your direction.
